specs2/src/main/scala/pathy/specs2/ValidateCodec.scala

```diff
@@ -17,29 +17,31 @@
 package pathy
 
 import slamdata.Predef._
-import pathy.scalacheck._
+import pathy.scalacheck._, Path._, PathyArbitrary._
```

Comment:
I'd move the `Path` import to its own line as it isn't part of `pathy.scalacheck`.